src/cephadm/cephadm

```diff
@@ -3555,8 +3565,8 @@ class CephadmAgent():
         self.ack = 1
         self.event = Event()
         self.mgr_listener = MgrListener(self)
-        self.ls_gatherer = AgentGatherer(self, self._get_ls, 'Ls')
-        self.volume_gatherer = AgentGatherer(self, self._ceph_volume, 'Volume')
+        self.ls_gatherer = AgentGatherer(self, self._get_ls, {}, 'Ls')
+        self.volume_gatherer = AgentGatherer(self, self._ceph_volume, {'enhanced': False}, 'Volume')
```

Comment:
```suggestion
        self.volume_gatherer = AgentGatherer(self, lambda: self._ceph_volume(enhanced=False), 'Volume')
```

Is there a need to keep arguments separately?

Reply:
no, just didn't think of use a lambda at the time